app/models/concerns/authorization_concern.rb

```diff
@@ -60,6 +65,17 @@ module AuthorizationConcern
 
     has_many :whitelisted_jwts
 
+    def expired?
+      return false unless roles.include?("hbx_staff") && EnrollRegistry.feature_enabled?(:admin_account_autolock)
```

Comment:
This isn't typically how we test for the presence of an admin role.

We usually check to see if the user has an associated person, then to see if that person has an associated and active hbx_staff_role.  There might even be a helper method I think.